src/sql/workbench/contrib/objectExplorer/browser/serverTreeView.ts

```diff
@@ -140,7 +141,9 @@ export class ServerTreeView extends Disposable implements IServerTreeView {
 				this._connectionManagementService.showConnectionDialog();
 			}));
 		}
-		this._tree = this._register(TreeCreationUtils.createRegisteredServersTree(container, this._instantiationService));
+
+		const horizontalScrollEnabled: boolean = this._configurationService.getValue(horizontalScrollingKey) || false;
```

Comment:
This won't react to settings changes will it?

Reply:
This setting requires a reload for the changes to show up (even for vscode trees)